docs/scenarios/index.md

```diff
@@ -1,8 +1,17 @@
 # Scenarios
 
+* [Authoring Solutions for .NET Core RC2](solution-authoring/index.md)
+    * [Building simple solutions with Visual Studio](solution-authoring/vs-golden-path.md)
```

Comment:
We should add language here if we have F# below. So "Building simple C# solutions with Visual Studio 2105". Please also put in VS version.